src/docs/sphinx/ARM.rst

```diff
@@ -4,6 +4,70 @@
 .. # SPDX-License-Identifier: MIT
 
 
-ARM Overview (under development)
+Arm Overview
 ================================
 
+This implementation of the Arm port of Variorum supports the Arm Juno r2 SoC.
+The Arm Juno r2 platform is a big.LITTLE cluster with Cortex-A72 (big) and
+Cortex-A53 (LITTLE) clusters (also called processors), respectively.
+It also has an Arm Mali GP-GPU. We have tested the Arm functionality of
+Variorum on Linaro supported Linux.
+
+References:
+
+* `Arm Juno r2 technical reference <https://developer.arm.com/documentation/100114/0200/>`_
+
+Monitoring and Control Through Sysfs Interface
+-----------------------------------------------
+
+The built-in monitoring interface on the Arm Juno r2 board is implemented
+by the on-board FPGA. Since this interface is not universally available
+on most Arm implementations, we leverage the standard Linux sysfs interface
+for monitoring and control.
+Following lists the specific metrics that are monitored on Arm Juno r2:
+
+* Power telemetry:
```

Comment:
Did you want this to be a subsection title instead of a new bullet? If so you can do:
```
Power telemetry:
^^^^^^^^^^^^^^^^^
```